gen/installer/util.py

```diff
@@ -43,9 +40,10 @@ def do_bundle_onprem(extra_files, gen_out, output_dir):
     for filename in extra_files:
         shutil.copy(filename, output_dir + filename)
 
-    # Copy the cluster packages
-    for name, info in gen_out.cluster_packages.items():
-        copy_makedirs(info['filename'], output_dir + info['filename'])
+    # Copy the config packages
+    for package_name in json.loads(gen_out.arguments['config_package_names']):
```

Comment:
nit: should really make all of these atomic copy out operations like fetch_bootstrap is...